src/cli.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { version } = require('../package.json');
+const ConfigSource = require('./config.js');
+const validate = require('../src/index.js');
+const { printConfig, printSummary, resolveFiles, printReport, abort } = require('./nodeUtils');
+const nodeLoader = require('./loader/node');
+const { getSummary, normalizePath } = require('./utils');
+const lint = require('./lint');
+
+
+async function run() {
+	console.log(`STAC Node Validator v${version}`);
+	console.log();
+
+	// Read config from CLI and config file (if any)
+	let config = ConfigSource.fromCLI();
+	if (typeof config.config === 'string') {
+		Object.assign(config, await ConfigSource.fromFile(config.config));
+	}
+	if (!config.loader) {
+		config.loader = nodeLoader;
+	}
+
+	// Handle ignoreCerts option in Node
+	if (config.ignoreCerts) {
+		process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
+	}
+
+	// Abort if no files have been provided
+	if (config.files.length === 0) {
+		abort('No path or URL specified.');
+	}
+
+	config.depth = config.depth >= 0 ? config.depth : -1;
+
+	// Verify files exist / read folders
+	const files = await resolveFiles(config.files, config.depth);
+	delete config.files;
+	for (let file in files.error) {
+		const error = files.error[file];
+		console.warn(`${file}: Can't be validated for the following reason: ${error}`);
+	}
+	if (files.files.length === 0) {
+		abort('No files found that are suitable for validation.');
+	}
+	else if (files.files.length === 1) {
+		data = data[0];
```

Comment:
I get an error here. Should it be:
```
data = files.files[0]
```
?

Reply:
Yes, not sure what happened there. I should enable tests again. Thanks.

```suggestion
		data = files.files[0];
```